### DISCUSSION TITLE
Welcome to Storybook Nuxt Discussions!

### DISCUSSION THREAD

**Discussion by chakAs3** [storybook-vue/nuxt#35] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rezkileap** [comment]
Are there going to be an upgrade script like storybook upgrade script (npx storybook@latest upgrade)?

**Reply by chakAs3** [reply]
@rezkileap yes definitely i'm just waiting to stabilize it the package first then, `npx storybook@latest init` and `npx storybook@latest upgrade` will be shipped with storybook core

**Comment by ajeetchaulagain** [comment]
Hi there,
Wondering if there is some sort of configuration to only run nuxt dev server while running `nuxt dev` ? Running `nuxt dev`  seems to spinning up both storybook and nuxt dev server.

**Reply by chakAs3** [reply]
remove the the storybook module from config, 
if you enable the module on you nuxt.config , storybook will automatically run

**Reply by ajeetchaulagain** [reply]
Thanks @chakAs3 for the insight. Removing module on nuxt.config seems to do a job :)
